### PR TITLE
.NET Interactive: Add_try_dotnet_to_batch_44b

### PR DESCRIPTION
**Pull request by v-mepa**
@mairaw to review!

## Add_try_dotnet_to_batch_44b

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=OneNote.File>
<meta name=Generator content="Microsoft OneNote 15">
</head>

<body lang=en-US style='font-family:Calibri;font-size:11.0pt'>

<div style='direction:ltr'>

https://docs.microsoft.com/en-us/dotnet/api/system.globalization.hijricalendar.getmonth | 45 | 24 | Batch 44 |  
-- | -- | -- | -- | --
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.calendaralgorithmtype | 42 | 7 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.textinfo.tolower | 42 | 16 | Batch 44 | Duplicate samples for some   overloads? Potentially move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.regioninfo.name | 40 | 12 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.clone | 39 | 5 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.clone | 36 | 11 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.currencygroupsizes | 35 | 0 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.gregoriancalendar.-ctor | 33 | 9 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.calendar.addyears | 31 | 12 | Batch 44 |  
https://docs.microsoft.com/en-us/dotnet/api/system.globalization.japanesecalendar.getera | 29 | 3 | Batch 44 |  

</div>

</body>

</html>

### CODE REVIEW

File: xml/System.Globalization/Calendar.xml [hunk 1]
```diff
@@ -842,7 +842,7 @@
   
  The <xref:System.DateTime.Kind%2A> property of the returned <xref:System.DateTime> value always equals <xref:System.DateTimeKind.Unspecified?displayProperty=nameWithType>. You can preserve the <xref:System.DateTime.Kind%2A> property of the `time` parameter by calling the <xref:System.DateTime.SpecifyKind%2A?displayProperty=nameWithType> method, as the following example shows.  
   
- :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.globalization.calendar.addmethods/cs/add1.cs" id="Snippet8":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.globalization.calendar.addmethods/cs/add1.cs" interactive="try-dotnet-method" id="Snippet8":::
```
Comment (mairaw) [inline xml/System.Globalization/Calendar.xml:845]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.globalization.calendar.addmethods/cs/add1.cs"  id="Snippet8":::
```

**Review by mairaw — APPROVED** [review]
LGTM. Let's merge after reviewing the build with a tiny change I made.

**Comment by v-mepa** [comment]
Yoooohoooo all 1000 done :)

**Comment by mairaw** [comment]
Yesssssss

**Comment by BillWagner** [comment]
Woot!  Awesome job! 🎆 🥂

**Pull request merged**